src/codemodder/codemods/xml_transformer.py

```diff
@@ -190,36 +191,27 @@ def apply(
                 parser.parse(file_context.file_path)
                 changes = transformer_instance.changes
                 output_file.seek(0)
+            except Exception:
+                file_context.add_failure(
+                    file_path, reason := "Failed to parse XML file"
+                )
+                logger.exception("%s %s", reason, file_path)
+                return None
 
-            except SAXParseException:
```

Comment:
specific is better than general but I chose to switch to the general exception to more closely follow libcst transformer